style.css

```diff
@@ -226,3 +226,78 @@ button:disabled {
 .chevron {
   margin: 0 30px;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  z-index: -2;
```

Comment:
- [ ] Kindly remove the `z-index` property because I can't access form inputs. and this will cause the background image not to display. Please kindly fix this issue also.